style_editor.html

```diff
@@ -9679,17 +9691,17 @@
       <span class="variant-alt-container">
         <span class="variant-label">Variant:</span>
         <span class="variant-alt-controls">
+          <input
```

Comment:
Just moving the decrease button next to value instead of far off the left side of the slider.